types.nix

```
@@ -655,7 +655,7 @@ rec {
         type = types.functionTo diskoLib.jsonType;
         default = dev: {
           fs.${dev} = ''
-            if ! $(swapon --show | grep -q '^${dev} '); then
+            if ! (swapon --show | grep -q '^${dev} '); then
```

Should this not also work?

```suggestion
            if ! swapon --show | grep -q '^${dev} '; then
```

---

Ah yeah, so it does! I'll push an update